schema/tables/screenlock.yml

```diff
@@ -0,0 +1,7 @@
+name: screenlock
+description: >- 
+  macOS screenlock status
+notes: >- 
+  - Only fetches results for osquery's current logged-in user context. If osquery is running as root, this table will return no results. 
```

Comment:
osquery running as root is almost always the case (haven't heard of non-root osquery deployments), so the table is currently unusable by Fleet users as is.

Reply:
Do you mean osquery deployments in general or only osquery deployments via fleet? (i.e. via fleetd)

Reply:
From @zwass:
> In practice it can be done to run as non-root, but I've never really seen it done and we definitely haven't documented doing that way.